pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -0,0 +1,12 @@
+---
+description: Consume Pyth Network prices in EVM applications
+---
+
+# Pyth on EVM
+
+On-chain EVM programs can use the [Solidity SDK](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/solidity) to read Pyth prices.
+The [EVM API reference](../../evm) lets you interactively explore the complete API of the Pyth contract.
+
+The off-chain portion of the application can use [pyth-evm-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js) to generate price update transactions. This repository's [Quickstart](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js#quickstart) includes an example of both the on- and off-chain code necessary to integrate with Pyth.
+
+## Examples
```

Comment:
We have an example for EVM. can you add it?
https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/examples/oracle_swap